mmv1/products/looker/Instance.yaml

```diff
@@ -401,25 +401,29 @@ properties:
     type: Enum
     description: |
       Platform editions for a Looker instance. Each edition maps to a set of instance features, like its size. Must be one of these values:
-      - LOOKER_CORE_TRIAL: trial instance (Currently Unavailable)
       - LOOKER_CORE_STANDARD: pay as you go standard instance (Currently Unavailable)
       - LOOKER_CORE_STANDARD_ANNUAL: subscription standard instance
       - LOOKER_CORE_ENTERPRISE_ANNUAL: subscription enterprise instance
       - LOOKER_CORE_EMBED_ANNUAL: subscription embed instance
       - LOOKER_CORE_NONPROD_STANDARD_ANNUAL: nonprod subscription standard instance
       - LOOKER_CORE_NONPROD_ENTERPRISE_ANNUAL: nonprod subscription enterprise instance
       - LOOKER_CORE_NONPROD_EMBED_ANNUAL: nonprod subscription embed instance
+      - LOOKER_CORE_TRIAL_STANDARD: A standard trial edition of Looker (Google Cloud core) product.
+      - LOOKER_CORE_TRIAL_ENTERPRISE: An enterprise trial edition of Looker (Google Cloud core) product.
+      - LOOKER_CORE_TRIAL_EMBED: An embed trial edition of Looker (Google Cloud core) product.
     immutable: true
     default_value: "LOOKER_CORE_TRIAL"
     enum_values:
-      - 'LOOKER_CORE_TRIAL'
```

Comment:
This is technically backwards incompatible, even though the API probably rejects this by now- let's keep the value for now, and clean it up as part of the major release . Removing from docs is fine atm.

Our contribution window is open now and will be for a couple more weeks; you'd just do this removal against the major release branch (https://googlecloudplatform.github.io/magic-modules/breaking-changes/make-a-breaking-change/#make-the-change-on-feature-branch-major-release-700) and add a brief note to the upgrade guide (https://googlecloudplatform.github.io/magic-modules/breaking-changes/make-a-breaking-change/#in-the-700-major-release). Most other notes on that page are for larger changes/removals which need more prep than an enum value.